### ISSUE TITLE
1.6.0-alpha.11 on Mac M2 OSX 13.4 comictagger startup error

### ISSUE THREAD

**Issue by BenRoe** [comictagger/comictagger#605]
I get the following error if i try to start the gui.
<details>
  <summary>Error message</summary>
```
Unhandled exception in app:
Traceback (most recent call last):
File "comictaggerlib/gui.py", line 136, in open_tagger_window
File "comictaggerlib/taggerwindow.py" line 90. in init
File "PyQt5/uic/__init__py", line 241, in loadUi
File "PyQt5/uic/Loader/loader.py", line 66, in loadui
File "PyQt5/uic/uiparser.py", line 1037, in parse
File "PyQt5/uic/uiparser.py", line 828, in createUserinterface
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 273, in createWidget
File "PyQt5/uic/uparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 503, in createLayout
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 546, in handleItem
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 273, in createWidget
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 273, in createWidget
File "PyQt5/uic/uparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 503, in createLayout
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "pvQt5/uic/uiparser.py", line 546, in handleItem
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 503, in createLavout
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 546, in
handleltem
File "PyQt5/uic/uparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 503, in createLavout
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 546, in handleItem
File "PyQt5/uic/uiparser.py", line 806, in traverseWidgetTree
File "PyQt5/uic/uiparser.py", line 264, in createWidget
File "PyQt5/uic/uiparser.py", line 228, in setupObject
File "PyQt5/uic/objcreator.py", line 100, in createQObject
File "PvQt5/uic/obicreator.py", line 125, in findQObjectType
File "PyQt5/uic/Loader/gobjectcreator.py" line 113, in search
ModuleNotFoundError: No module named
'comictaggerlib.ui.customwidgets'
```
</details>

![1 6 0-alpha 11-mac-m2](https://github.com/comictagger/comictagger/assets/643834/4afc126d-1f12-415e-8e36-4925a8a8c550)

**Comment by AusNaminator** [comment]
I am experiencing the same error on Windows

**Comment by mizaki** [comment]
For now, if you wish to use alpha 11 rather than going back to alpha 10: Copy https://github.com/comictagger/comictagger/blob/develop/comictaggerlib/ui/customwidgets.py to the expected location (`_internal/comictaggerlib/ui/` in the linux package for example) and it should run.

This might only work on linux...

**Comment by BenRoe** [comment]
@mizaki thanks for the hint.
I copied [customwidgets.py] (https://github.com/comictagger/comictagger/blob/develop/comictaggerlib/ui/customwidgets.py) to `/Applications/ComicTagger.app/Contents/Resources/comictaggerlib/ui/`.

Still get the same error.

**Comment by lordwelch** [comment]
This is fixed by 4a33dbde465a35ef215fb095518eb0edd8d9f2de

**Comment by lordwelch** [comment]
Fixed in [1.6.0-alpha.12]

[1.6.0-alpha.12]: https://github.com/comictagger/comictagger/releases/1.6.0-alpha.12
